examples_tests/18.MitsubaLoader/main.cpp

```diff
@@ -35,12 +35,12 @@ vec3 irr_computeLighting(inout irr_glsl_IsotropicViewSurfaceInteraction out_inte
 
 	irr_glsl_BSDFIsotropicParams params;
 	vec3 color = vec3(0.0);
-	for (int i = 0; i < 13; ++i)
+	for (int i = 0; i < 132; ++i)
 	{
 		SLight l = lights[i];
 		vec3 L = l.position-WorldPos;
 		params.L = L;
-		color += irr_bsdf_cos_eval(params, out_interaction, dUV)*l.intensity*0.01 / dot(L,L);
+		color += irr_bsdf_cos_eval(params, out_interaction, dUV)*l.intensity*4.0 / dot(L,L);
```

Comment:
you can get a good approximation of intensity of an area light by doing the following:
1) compute light surface area (we have functions for that used in ex 21)
2) multiply by radiance and 2PI

Reply:
done?

Reply:
yes, it was always like that but still sometimes i have to use extra factors

Reply:
actually only in ditt office scene it looked nice with factor of 1.0 so far

Reply:
ok, might need to slap the autoexposure extension on the example then

Reply:
yea ive wanted to make use of autoexposure in mitsuba loader example but its just lowest priority